### PR TITLE
Add player to shipbullet, and fix point adding <feature

### PR DESCRIPTION
**Pull request by bryanvanwijk**
I made it possible to see from which player a bullet is so the points can be added to the right player in a multi player game.

### CODE REVIEW

**Comment by jochem725** [comment]
Why does the SpaceShip class no longer implement ShootingUnit? It is a Shooting Unit.
Maybe there is another way to obtain the player connected to the spaceship so this interface can remain intact.

**Comment by bryanvanwijk** [comment]
Good point I changed it.

**Comment by jochem725** [comment]
Great! I'll agree with merging now.

File: src/main/java/spaceinvaders/group_22/Player.java [hunk 1]
```diff
@@ -144,5 +148,13 @@ public final void addLife() {
 	public final void setActivePowerUps(final ArrayList<PowerUp> newactivePowerUps) {
 		this.activePowerUps = newactivePowerUps;
 	}
+	/**
+	 * Let this player shoot a bullet.
+	 * @param bulletVelx speed of the bullet
+	 * @return Bullet that is shot.
+	 */
+	public final ShipBullet shootBullet(final double bulletVelx) {
```
Comment (doriandekoning) [inline src/main/java/spaceinvaders/group_22/Player.java]: Why does a player shoot a bullet all of the sudden, this seems nog logical to me and conflicts with the responsibility of the player class. I think player.getSpaceShip().shootBullet() would be better.

**Comment by doriandekoning** [comment]
Apart from that one comment it seems fine. If you have a good argument against it or change you can merge this branch.

**Comment by bryanvanwijk** [comment]
I removed this responsibility from the player class.

**Pull request merged**